docs/developers/add-new-bidder.md

```diff
@@ -33,6 +33,16 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
+If bidder is going to support long form video make sure bidder returns:
+
+- category in `bid.bidVideo.PrimaryCategory` or `TypedBid.bid.Cat` field. <br>
+Category from `bid.bidVideo.PrimaryCategory` field should be a string value, translated to Primary ad server format.<br>
+Category from `TypedBid.bid.Cat` field should be a single string value in string array in IAB format.
+To learn more about IAB categories, please refer to this convenience link (not the final official definition): [IAB categories](https://adtagmacros.com/list-of-iab-categories-for-advertisement/)
```

Comment:
1) Can we fix the identation of these 3 sentences? Can they be in one paragraph?

Reply:
2) Can we please remove HTML `<br>` tags?
3) In the sentence "Category from `bid.bidVideo.PrimaryCategory` field should be a string value, translated to Primary ad server format", what is the "Primary add server format"?
4) The following sentence: "category in `bid.bidVideo.PrimaryCategory` or `TypedBid.bid.Cat` field. <br>" could be interpreted as: "Field, `bid.bidVideo.PrimaryCategory` can also be called `TypedBid.bid.Cat`". Can we rephrase?